platform/ui/src/views/StudyList/components/StudyListTable.js

```diff
@@ -170,27 +185,85 @@ const TableRow = () => {
   );
 };
 
-const StudyListTable = () => {
+TableRow.propTypes = {
+  AccessionNumber: PropTypes.string.isRequired,
+  Modalities: PropTypes.string.isRequired,
+  Instances: PropTypes.number.isRequired,
+  PatientId: PropTypes.string.isRequired,
+  PatientName: PropTypes.string.isRequired,
+  StudyDescription: PropTypes.string.isRequired,
+  StudyDate: PropTypes.string.isRequired,
+  series: PropTypes.array.isRequired,
+};
+
+const StudyListTable = ({ studies, numOfStudies }) => {
+  const renderTable = () => {
+    return (
+      <table className="w-full text-white">
+        <tbody>
+          {studies.map((study, i) => (
+            <TableRow
+              key={i}
+              AccessionNumber={study.AccessionNumber || ''}
+              Modalities={study.Modalities || ''}
+              Instances={study.Instances || ''}
+              StudyDescription={study.StudyDescription || ''}
+              PatientId={study.PatientId || ''}
+              PatientName={study.PatientName || ''}
+              StudyDate={study.StudyDate || ''}
+              series={study.series || []}
+            />
+          ))}
+        </tbody>
+      </table>
+    );
+  };
+
+  const renderEmpty = () => {
+    return (
+      <div className="flex flex-col items-center justify-center pt-48">
+        <Icon name="magnifier" className="mb-4" />
+        <Typography className="text-custom-aquaBright" variant="h5">
+          No studies available
+        </Typography>
+      </div>
+    );
+  };
+
   return (
     <>
       <div className="bg-black">
         <div className="container m-auto relative">
-          <div className="bg-custom-blue text-center text-base py-1 rounded-b sticky top-0">
-            <p className="text-white">
-              Filter list to 100 studies or less to enable sorting
-            </p>
-          </div>
-          <table className="w-full text-white">
-            <tbody>
-              {new Array(30).fill('').map((empty, i) => (
-                <TableRow key={i} />
-              ))}
-            </tbody>
-          </table>
+          {numOfStudies > 100 && (
+            <div className="bg-custom-blue text-center text-base py-1 rounded-b sticky top-0">
+              <p className="text-white">
+                Filter list to 100 studies or less to enable sorting
+              </p>
+            </div>
+          )}
```

Comment:
This will probably be a render prop in finalized version. The ">100" constraint only applies to DICOMWeb server sources, so the parent may use this as a conditional target to add an "is loading" message, or a display a "banner notification".

Something we can discuss, but hesitant to hardcode "100" studies here.